2022-07-30 13:56:37,043 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-07-30 13:56:37,045 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\diedi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\instabot\bot\bot.py
2022-07-30 13:56:37,046 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-07-30 13:56:37,341 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2022-07-30 13:56:37,582 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2022-07-30 13:56:37,793 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2022-07-30 13:56:38,019 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2022-07-30 13:56:41,319 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2022-07-30 13:56:41,319 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXHKI0QCsaIIVfMrQ07kE0hvW8AzcZlbPrF67KgNWAL_HXNbf_Q1-0nG9kCBviCGxghz0eA/Afxij3id0yfmT6PT5Kii0h-8rY1LjFFAcCS8Fxo1Ofh1OiyQKJdsM8fJjkUvAf2GUaTzaxS3r881jA/","api_path":"/challenge/AXHKI0QCsaIIVfMrQ07kE0hvW8AzcZlbPrF67KgNWAL_HXNbf_Q1-0nG9kCBviCGxghz0eA/Afxij3id0yfmT6PT5Kii0h-8rY1LjFFAcCS8Fxo1Ofh1OiyQKJdsM8fJjkUvAf2GUaTzaxS3r881jA/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2022-07-30 13:56:41,319 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-07-30 13:56:41,320 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2022-07-30 13:56:41,321 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
